source/PrintDiamond.cpp

```diff
@@ -0,0 +1,86 @@
+#include "PrintDiamond.h"
+
+int answer()
+{
+	return 6 * 9;
+}
+
+void PrintDiamondClass::InitArray()
+{
+	m_alphabet = { { 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 
+					 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z' } };
+	for (int i = 0; i <= 25; i++)
+	{
+		m_numbers[i] = i + 1;
+	}
+
+	for (int i = 0; i <= 51; i++)
+	{
+		for (int i2 = 0; i2 <= 51; i2++)
+		{
+			m_diamond_grid[i][i2] = ' ';
+		}
+	}
+}
+
+void PrintDiamondClass::LetterToNum()
```

Comment:
Are you aware that a character is just another integer? See https://en.wikipedia.org/wiki/ASCII for reference.
Basically, it is perfectly legal to do something like this:
`int x = 'A' - 73;`